### PR TITLE
Add version information to service metadata

### PR DESCRIPTION
**Pull request by phynes-sensiblecode**

### CODE REVIEW

**Review by SpencerHedger — CHANGES_REQUESTED** [review]
There is nothing I could find functionally "wrong" in here, but I have a few suggestions/thoughts.

File: CENSUS_METADATA.md [hunk 1]
```diff
@@ -100,6 +99,21 @@ message.
 | Field | GraphQL Type | Source (en) | Source (cy) |
 | --- | --- | --- | --- |
 | `description` | `String!` | "Census 2021 metadata" | "Census 2021 metadata in Welsh" |
+| `version` | `Version!` | Version information from various sources |  |
+
+## Version
```
[the remark this answers is not in the document]
Comment (phynes-sensiblecode) [reply CENSUS_METADATA.md:104]: After a discussion with other team members, we've opted for Spencer's suggestion and will add further information about the build.

This renders some of the other discussions obsolete.

File: test/expected/service-metadata-best-effort.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "lang": "en",
+        "meta": {
+            "description": "Census 2021 metadata",
+            "build": {
```
Comment (SpencerHedger) [inline test/expected/service-metadata-best-effort.json:6]: Maybe a little late in the day to raise this thought, but is `build` safe to use and unlikely to clash with any other properties that might be needed by ONS? My only other suggestion would be `_build` - but I'm thinking we are probably unlikely to have a clash, what do you think?

**Review by SpencerHedger — CHANGES_REQUESTED** [review]
Looks good, couple of small thoughts/suggestions but nothing major.

File: CENSUS_METADATA.md [hunk 3]
```diff
@@ -100,6 +99,33 @@ message.
...
+| `build` | `Build!` | Build and version information from various sources |  |
+
+## Build
+
+The data in `Build` is intended for debug purposes.
+It provides some information about the build process, where the CSV files were converted to JSON.
+
+| Field | GraphQL Type | Source (en) | Source (cy) |
+| --- | --- | --- | --- |
+| `created` | `String!` | Time in ISO 8601 format that the Python CSV to JSON script was executed | |
```
[the remark this answers is not in the document]
Comment (phynes-sensiblecode) [reply CENSUS_METADATA.md:111]: All the metadata JSON files are built from this one script, so I don't think it adds much value.

File: test/expected/service-metadata-best-effort.json (hunk 2 above)
Comment (phynes-sensiblecode) [reply test/expected/service-metadata-best-effort.json:6]: We don't have any ONS specific information at the service level. I think we should be fine with `build`.

**Review by SpencerHedger — APPROVED** [review]
Looks good to me.

**Pull request merged**